spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/pom.xml

```diff
@@ -1,20 +1,62 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<artifactId>spring-cloud-data-module-deployer-cloudfoundry</artifactId>
-	<packaging>jar</packaging>
-	<name>spring-cloud-data-module-deployer-cloudfoundry</name>
-	<description>module deployer SPI implementation for Cloud Foundry</description>
-	<parent>
-		<groupId>org.springframework.cloud</groupId>
-		<artifactId>spring-cloud-data-module-deployers-parent</artifactId>
-		<version>1.0.0.BUILD-SNAPSHOT</version>
-	</parent>
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-data-module-deployer-spi</artifactId>
-		</dependency>
-	</dependencies>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-cloud-data-module-deployer-cloudfoundry</artifactId>
+    <packaging>jar</packaging>
+    <name>spring-cloud-data-module-deployer-cloudfoundry</name>
+    <description>module deployer SPI implementation for Cloud Foundry</description>
+
+    <parent>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-data-module-deployers-parent</artifactId>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <cloudfoundry-client-lib.version>1.1.3</cloudfoundry-client-lib.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-data-module-deployer-spi</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.security.oauth</groupId>
+            <artifactId>spring-security-oauth2</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+    </dependencies>
+
+    <repositories>
```

Comment:
These two repos should be removed (and build with `mvn -s .settings.xml` until we add maven wrapper support)